smoke-tests-otel-starter/spring-smoke-testing/build.gradle.kts

```diff
@@ -2,7 +2,7 @@ import org.springframework.boot.gradle.tasks.bundling.BootJar
 
 plugins {
   id("otel.java-conventions")
-  id("org.springframework.boot") version "2.7.18"
+  id("org.springframework.boot") version "2.6.15"
```

Comment:
why are you changing the version here, won't renovate immediately change it back?

Reply:
will add to renovate rules